config/country_codes.yml

```diff
@@ -1,5 +1,253 @@
 ---
 iso:
   3166:
-    regex:
-      ^(AF|AX|AL|DZ|AS|AD|AO|AI|AQ|AG|AR|AM|AW|AU|AT|AZ|BS|BH|BD|BB|BY|BE|BZ|BJ|BM|BT|BO|BQ|BA|BW|BV|BR|IO|BN|BG|BF|BI|KH|CM|CA|CV|KY|CF|TD|CL|CN|CX|CC|CO|KM|CG|CD|CK|CR|CI|HR|CU|CW|CY|CZ|DK|DJ|DM|DO|EC|EG|SV|GQ|ER|EE|ET|FK|FO|FJ|FI|FR|GF|PF|TF|GA|GM|GE|DE|GH|GI|GR|GL|GD|GP|GU|GT|GG|GN|GW|GY|HT|HM|VA|HN|HK|HU|IS|IN|ID|IR|IQ|IE|IM|IL|IT|JM|JP|JE|JO|KZ|KE|KI|KP|KR|KW|KG|LA|LV|LB|LS|LR|LY|LI|LT|LU|MO|MK|MG|MW|MY|MV|ML|MT|MH|MQ|MR|MU|YT|MX|FM|MD|MC|MN|ME|MS|MA|MZ|MM|NA|NR|NP|NL|NC|NZ|NI|NE|NG|NU|NF|MP|NO|OM|PK|PW|PS|PA|PG|PY|PE|PH|PN|PL|PT|PR|QA|RE|RO|RU|RW|BL|SH|KN|LC|MF|PM|VC|WS|SM|ST|SA|SN|RS|SC|SL|SG|SX|SK|SI|SB|SO|ZA|GS|SS|ES|LK|SD|SR|SJ|SZ|SE|CH|SY|TW|TJ|TZ|TH|TL|TG|TK|TO|TT|TN|TR|TM|TC|TV|UG|UA|AE|GB|US|UM|UY|UZ|VU|VE|VN|VG|VI|WF|EH|YE|ZM|ZW)$
+    codes:
+    - "AD"
+    - "AE"
+    - "AF"
+    - "AG"
+    - "AI"
+    - "AL"
+    - "AM"
+    - "AO"
+    - "AQ"
+    - "AR"
+    - "AS"
+    - "AT"
+    - "AU"
+    - "AW"
+    - "AX"
+    - "AZ"
+    - "BA"
+    - "BB"
+    - "BD"
+    - "BE"
+    - "BF"
+    - "BG"
+    - "BH"
+    - "BI"
+    - "BJ"
+    - "BL"
+    - "BM"
+    - "BN"
+    - "BO"
+    - "BQ"
+    - "BR"
+    - "BS"
+    - "BT"
+    - "BV"
+    - "BW"
+    - "BY"
+    - "BZ"
+    - "CA"
+    - "CC"
+    - "CD"
+    - "CF"
+    - "CG"
+    - "CH"
+    - "CI"
+    - "CK"
+    - "CL"
+    - "CM"
+    - "CN"
+    - "CO"
+    - "CR"
+    - "CU"
+    - "CV"
+    - "CW"
+    - "CX"
+    - "CY"
+    - "CZ"
+    - "DE"
+    - "DJ"
+    - "DK"
+    - "DM"
+    - "DO"
+    - "DZ"
+    - "EC"
+    - "EE"
+    - "EG"
+    - "EH"
+    - "ER"
+    - "ES"
+    - "ET"
+    - "FI"
+    - "FJ"
+    - "FK"
+    - "FM"
+    - "FO"
+    - "FR"
+    - "GA"
+    - "GB"
+    - "GD"
+    - "GE"
+    - "GF"
+    - "GG"
+    - "GH"
+    - "GI"
+    - "GL"
+    - "GM"
+    - "GN"
+    - "GP"
+    - "GQ"
+    - "GR"
+    - "GS"
+    - "GT"
+    - "GU"
+    - "GW"
+    - "GY"
+    - "HK"
+    - "HM"
+    - "HN"
+    - "HR"
+    - "HT"
+    - "HU"
+    - "ID"
+    - "IE"
+    - "IL"
+    - "IM"
+    - "IN"
+    - "IO"
+    - "IQ"
+    - "IR"
+    - "IS"
+    - "IT"
+    - "JE"
+    - "JM"
+    - "JO"
+    - "JP"
+    - "KE"
+    - "KG"
+    - "KH"
+    - "KI"
+    - "KM"
+    - "KN"
+    - "KP"
+    - "KR"
+    - "KW"
+    - "KY"
+    - "KZ"
+    - "LA"
+    - "LB"
+    - "LC"
+    - "LI"
+    - "LK"
+    - "LR"
+    - "LS"
+    - "LT"
+    - "LU"
+    - "LV"
+    - "LY"
+    - "MA"
+    - "MC"
+    - "MD"
+    - "ME"
+    - "MF"
+    - "MG"
+    - "MH"
+    - "MK"
+    - "ML"
+    - "MM"
+    - "MN"
+    - "MO"
+    - "MP"
+    - "MQ"
+    - "MR"
+    - "MS"
+    - "MT"
+    - "MU"
+    - "MV"
+    - "MW"
+    - "MX"
+    - "MY"
+    - "MZ"
+    - "NA"
+    - "NC"
+    - "NE"
+    - "NF"
+    - "NG"
+    - "NI"
+    - "NL"
+    - "NO"
```

Comment:
because of norway, i had to quote the yaml values, as `- NO` gets interpreted to `false` ;-)